pkgs/tools/misc/digitemp/default.nix

```diff
@@ -0,0 +1,57 @@
+{ fetchFromGitHub, lib, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "digitemp";
+  version = "3.7.2";
+
+  src = fetchFromGitHub {
+    owner = "bcl";
+    repo = "digitemp";
+    rev = "v${version}";
+    sha256 = "19zka5fcdxhhginaspak76l984iqq9v2j6qrwvi5mvca7bcj8f72";
+  };
+
+  enableParallelBuilding = true;
+
+  preBuild = ''
```

Comment:
you don't need this. when building in a sandbox you can get a clean environment.

Reply:
I originally just copied that from the gentoo ebuild... But I think the only reason it was there was the conditional linking against libusb (the part which is broken), and I omitted that adapter interface completely from this derivation... It might be added later, but I think if it gets added later, it should be added as a separate derivation, because it doesn't share compilation artifacts with the other adapter targets. (because of different `CFLAGS` and `LIBS`)